lib/screen/login_page.dart

```diff
@@ -0,0 +1,152 @@
+                  shape: BoxShape.circle,
+                  image: DecorationImage(
+                    image: AssetImage(SochemIcon),
+                    fit: BoxFit.scaleDown,
+                  ),
+                ),
+              ),
+
+              //Buttons for login options
+              Container(
+                padding: EdgeInsets.symmetric(
+                    vertical: MediaQuery.of(context).size.height * 0.1),
+                child: Column(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(bottom: 25),
+                      child:
+
+                          // Login button
+                          ElevatedButton(
+                        onPressed: () {},
+                        child: Text(
+                          "LOGIN",
+                          style: TextStyle(color: Colors.blue),
+                        ),
+                        style: ElevatedButton.styleFrom(
+                          primary: Colors.white,
+                          textStyle: TextStyle(
+                            fontSize: 20,
+                            color: Colors.blue,
+                          ),
+                          fixedSize: Size(
+                              MediaQuery.of(context).size.width * 0.78, 50),
+                          shape: RoundedRectangleBorder(
+                            borderRadius: BorderRadius.circular(10),
+                          ),
+                        ),
+                      ),
+                    ),
+
+                    // Guest Buttton
+                    TextButton(
+                      onPressed: () {},
+                      child: Text('Guest'),
+                      style: TextButton.styleFrom(
+                        primary: Colors.white,
+                        elevation: 0,
+                        textStyle: TextStyle(
+                          fontSize: 20,
+                        ),
+                        fixedSize:
+                            Size(MediaQuery.of(context).size.width * 0.78, 50),
+                        side: BorderSide(color: Colors.white, width: 1),
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(10),
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+// classes
+class Bluepainter extends CustomPainter {
+  @override
+  void paint(Canvas canvas, Size size) {
+    final height = size.height;
+    final width = size.width;
+    final midx = width * 0.5;
+    final midy = height * 0.5;
+    Paint paint = Paint();
+
+    // Left triangle
+    Path triangle1 = Path();
+    triangle1.moveTo(0, 0);
+    triangle1.lineTo(midx, midy);
+    triangle1.lineTo(0, height);
+    triangle1.close();
+    paint.color = Color.fromARGB(255, 58, 110, 196);
```

Comment:
Include the colors like this in the form of constants that can be used in other files.

For eg - See the usage of `SochemIcon` constant